### ISSUE TITLE
A better background with better resolution

### ISSUE THREAD

**Issue by 3watermelons** [3watermelons/Banking-System#11]
_A more professional site bg is needed_

**Comment by sano2019** [comment]
I can take this if possible :) Please assign to me

**Comment by Ishikasaraf20** [comment]
i can work on it please assign thhis to me

**Comment by 3watermelons** [comment]
@sano2019 @Ishikasaraf20 and other assignees

Add your bg to the code
Take screenshot of the site with 3 of your top bg designs
Attach in comment
Create PR with the bg that gets approved

**Comment by sano2019** [comment]
Below are 3 options. I kinda liked the design with the pig so I looked for some alternatives for that with higher resolution. 

Is there any size limit you want to set? I have used the full size backgrounds, so that puts them between 464kb for the smallest (white with pig) and 1.7mb for the largest (the pig sniffing the money).

Let me know what your preference is and what you want to do with the size of the image if anything :)

<img width="1772" alt="Screenshot 2022-10-24 at 08 53 46" src="https://user-images.githubusercontent.com/53917461/197465340-45f37b16-3387-479d-b9f1-00ba15668218.png">
<img width="1776" alt="Screenshot 2022-10-24 at 08 53 25" src="https://user-images.githubusercontent.com/53917461/197465372-1780943f-90c1-4e66-a2b6-f309742f8738.png">
<img width="1774" alt="Screenshot 2022-10-24 at 08 52 56" src="https://user-images.githubusercontent.com/53917461/197465377-653a1c65-1994-4b13-918b-4aa572d4a80f.png">
